beancount_import/source/schwab_csv.py

```diff
@@ -766,6 +859,76 @@ def get_accounts(self) -> List[str]:
         return [self.get_primary_account(), self.get_other_account()]
 
 
+@dataclass(frozen=True)
+class Merger(TransactionEntry):
+    fees_account: str
+    symbol: str
+    quantity: Decimal
+    price: Optional[Decimal]
+    merger_spec: MergerSpecification
+    fees: Optional[Decimal]
+
+    def get_sub_account(self) -> Optional[str]:
+        return self.symbol
+
+    def get_other_account(self) -> str:
+        return f"{self.account}:{self.merger_spec.symbol}"
+
+    def get_postings(self) -> List[Posting]:
+        postings = [
+            Posting(
+                account=self.get_primary_account(),
+                units=Amount(self.quantity, currency=self.symbol),
+                cost=CostSpec(
+                    number_per=self.price,
+                    number_total=None,
+                    currency=CASH_CURRENCY,
+                    date=None,
+                    # at the moment requires manually choosing the lot
```

Comment:
I think this could be resolvable via `lot_info`, but it's OK to merge without implementing that yet.

Reply:
I want to handle this in a separate PR. This PR is big as is.